streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/config/OpenapiConfig.java

```diff
@@ -76,4 +76,12 @@ public GroupedOpenApi publicApiV1() {
   public GroupedOpenApi publicApiV2() {
     return GroupedOpenApi.builder().group("v2").pathsToMatch("/v2/**").build();
   }
+
+  private static final String OPEN_API_INFO_TITLE = "Apache StreamPark Api Docs";
+  private static final String OPEN_API_INFO_DESCRIPTION = "Apache StreamPark Api Docs";
```

Comment:
Its could be inlined into a new contant named `OPEN_API_INFO_OVERVIEW`, then remove the two lines, because the contents of the two lines is the same.

Reply:
I have modified